service-front/web/main.scss

```
@@ -12,3 +12,33 @@ $govuk-assets-path: "../assets/";
 .moj-banner--date {
   max-width: 150px;
 }
+
+.govuk-inset-text-blue {
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
+  padding: 15px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  clear: both;
+  border-left: 2px solid #1d70b8;
+  background-color: #f3f2f1;
```

Better to use variables and mixins from the GDS framework so that we match any changes:
```suggestion
  @include govuk-font($size: 16);
  color: $govuk-text-colour;
  padding: govuk-spacing(3);
  margin-top: govuk-spacing(4);
  margin-bottom: govuk-spacing(4);
  clear: both;
  border-left: 2px solid $govuk-brand-colour;
  background-color: govuk-colour("light-grey");
```